hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -297,19 +299,22 @@ private FlinkOptions() {
       .key("read.streaming.skip_compaction")
       .booleanType()
       .defaultValue(false)// default read as batch
-      .withDescription("Whether to skip compaction instants for streaming read,\n"
-          + "there are two cases that this option can be used to avoid reading duplicates:\n"
-          + "1) you are definitely sure that the consumer reads faster than any compaction instants, "
-          + "usually with delta time compaction strategy that is long enough, for e.g, one week;\n"
+      .withDescription("Whether to skip compaction instants and avoid reading compacted base files for streaming read to improve read performance.\n"
+          + "There are two cases that this option can be used to avoid reading duplicates:\n"
+          + "1) you are definitely sure that the consumer reads [faster than/completes before] any compaction instants "
+          + "when " + HoodieCompactionConfig.PRESERVE_COMMIT_METADATA.key() + " is set to false.\n"
           + "2) changelog mode is enabled, this option is a solution to keep data integrity");
```

Comment:
No matter whether `HoodieCompactionConfig.PRESERVE_COMMIT_METADATA.key()` is true or false, the base files would be skipped, check the latest changes for `IncrementalInputSplits` again for details.

Reply:
>  if read.streaming.skip_compaction=false and HoodieCompactionConfig.PRESERVE_COMMIT_METADATA.key()=false, will there be duplicated data?

Yes, it may introduce duplicates, let's make the doc more clear.

Reply:
What about this:

```text
Whether to skip compaction instants and avoid reading compacted base files for streaming read to improve read performance.

Please set this configuration to true to avoid reading duplicates if:
1) HoodieCompactionConfig.PRESERVE_COMMIT_METADATA.key() is set to false 
2) changelog mode is enabled, this option is a solution to keep data integrity
```

Is this better enough?